packages/client/src/context/Auth.context.tsx

```diff
@@ -106,28 +116,59 @@ export const AuthProvider: FC<AuthProviderProps> = ({ children }) => {
     handleUnauthenticated();
   };
 
+  // Handle switch tab
+  const [activeTab, setActiveTab] = useState<string>('login');
+  const handleTabClick = (tab:string) => {
+    setActiveTab(tab);
+  };
+
+  const location = useLocation();
+  const isResetPassword = location.pathname === "/reset-password";   // determine if it is reset password page
+
   return (
-    <AuthContext.Provider value={{ token, authenticated, decodedToken, logout }}>
-      {!authenticated && organization && (
-        <FirebaseLoginWrapper setToken={handleAuthenticated} organization={organization} />
-      )}
-      {authenticated && children}
-    </AuthContext.Provider>
+    <div style={styles.container}>
+      {!isResetPassword &&
+      (<div style={styles.tabContainer}>
+        <div style={{ ...styles.tab, ...(activeTab === 'login' && styles.activeTab) }} onClick={() => handleTabClick('login')}>Login</div>
+        <div style={{ ...styles.tab, ...(activeTab === 'signup' && styles.activeTab) }} onClick={() => handleTabClick('signup')}>Signup</div>
+      </div>)}
+      
+      {!isResetPassword && (<div className="options">
+        <label style={{display: 'block', textAlign: 'center' as 'center', marginBottom: '5px'}}>Organization</label>
+        <Select style={{ width: '300px', height: '40px', marginBottom: '20px'}}>
+          {organizationList?.map((organization, index) => {
+            return (
+              <MenuItem key={index} value={10}>
```

Comment:
In this case, the value should be the organization. Then the onchange of the select would be to set the organization to the value selected.